### DISCUSSION TITLE
HELP~~~Auth login authentication has been configured in Jaeger query, and the account password has been configured but has not taken effect

### DISCUSSION THREAD

**Discussion by xkz130** [jaegertracing/jaeger#4411] [Q&A]
The figure shows the configuration of my UI page， After the configuration is completed, the service is restarted, but when accessed through IP: 16686, it can still be directly accessed without the need to enter an account password

![image](https://user-images.githubusercontent.com/46994912/234233290-97af0d89-9e29-4595-8db0-1f58200b5180.png)
![image](https://user-images.githubusercontent.com/46994912/234233752-3cc3bf49-4887-428e-be20-76a7819697a9.png)

**Comment by yurishkuro** [comment]
Where do you see documentation saying that this is a supported configuration for Jaeger UI?

**Reply by xkz130** [reply]
Ask for the method found through chatgpt and search in/ I saw the description of the query ui configuration string in query Linux - h, so I think there should be a method to configure authentication login
![image](https://user-images.githubusercontent.com/46994912/234441899-ecf1f41b-4b7c-4a8b-a33c-471e07515775.png)

**Reply by yurishkuro** [reply]
https://www.jaegertracing.io/docs/1.44/faq/#how-do-i-configure-authentication-for-jaeger-ui

**Comment by xkz130** [comment]
And I want to know where the static files of this type of index-69a9fbe0.js are located in the jaeger query container. I didn't find them in the global search of the container, Have these files been directly compiled into binary
![image](https://user-images.githubusercontent.com/46994912/234443324-9ca49851-bd1c-4a39-ba9c-2f9984876333.png)
![image](https://user-images.githubusercontent.com/46994912/234443343-c1c44e6a-d652-4707-8d43-cf2ea6922c56.png)

**Reply by yurishkuro** [reply]
yes, they are embedded in the binary

**Reply by xkz130** [reply]
So, where can I get these static files? thank you

**Reply by yurishkuro** [reply]
jaeger-ui repo

**Reply by xkz130** [reply]
> jaeger-ui repo
I found that jaeger query compiles both the front-end and back-end into a binary file. Can this be compiled using source code to separate the front-end and back-end?
